### PR TITLE
Feature/tri 766 irs local deployment

### PR DESCRIPTION
**Pull request by ds-jhartmann**
https://jira.catena-x.net/browse/TRI-766
Added helm charts for local deployment

### CODE REVIEW

File: .github/workflows/int-setup-testdata.yml [hunk 1]
```diff
@@ -7,32 +7,16 @@ on:
...
-        required: true
-        type: string
-      submodelUrl2:
-        description: 'URL for Submodel server 2'
-        required: true
-        type: string
-      submodelUrl3:
-        description: 'URL for Submodel server 3'
+      submodelUrls:
+        description: 'URLs for Submodel servers'
```
Comment (ds-jkreutzfeld) [inline .github/workflows/int-setup-testdata.yml:11]: Do I have to separate multiple URLs with a comma, semicolon or space? A little support in the description would be good. :)

File: .github/workflows/int-setup-testdata.yml [hunk 2]
```diff
@@ -7,32 +7,16 @@ on:
...
-        required: true
-        type: string
-      edcUrl2:
-        description: 'Provider 2 control plane URL'
-        required: true
-        type: string
-      edcUrl3:
-        description: 'Provider 3 control plane URL'
+      edcUrls:
+        description: 'Provider control plane URLs'
```
Comment (ds-jkreutzfeld) [inline .github/workflows/int-setup-testdata.yml:19]: Do I have to separate multiple URLs with a comma, semicolon or space? A little support in the description would be good. :)

File: charts/irs-environments/local/Chart.yaml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 1.0.0
+
+dependencies:
+  - name: irs-helm
+    repository: https://catenax-ng.github.io/product-item-relationship-service
+    version: 2.1.0
```
Comment (ds-jkreutzfeld) [inline charts/irs-environments/local/Chart.yaml:22]: We can update this to 2.2.0 now.

File: README.md [hunk 4]
```diff
@@ -43,6 +43,31 @@ The following subsection provides instructions for running the infrastructure on
 - API docs: http://localhost:8080/api/api-docs
 - API docs in yaml:  http://localhost:8080/api/api-docs.yaml
 
+### Local Helm deployment
+
+#### Prerequisites
+
+- install rancher desktop
+- install kubectl
+- Set Credentials for external environments in [helm charts](charts/irs-environments/local/values.yaml)
```
Comment (ds-jkreutzfeld) [inline README.md:52]: Maybe you can explain which config entries need to be set so the user does not need to search the file :)

File: .github/workflows/int-setup-testdata.yml (hunk 1 above)
Comment (ds-jhartmann) [reply .github/workflows/int-setup-testdata.yml]: Done

File: .github/workflows/int-setup-testdata.yml (hunk 2 above)
Comment (ds-jhartmann) [reply .github/workflows/int-setup-testdata.yml]: Done

File: charts/irs-environments/local/Chart.yaml (hunk 3 above)
Comment (ds-jhartmann) [reply charts/irs-environments/local/Chart.yaml]: Done

File: README.md (hunk 4 above)
Comment (ds-jhartmann) [reply README.md]: Done

**Review by ds-jkreutzfeld — APPROVED** [review]

**Pull request merged**